modules/ROOT/nav.adoc

```diff
@@ -60,6 +60,13 @@
 ** xref:tmt/video.adoc[Videos]
 
 * Tools
+** IO-Tool
+*** xref:tools/io-tool/overview.adoc[Overview]
+*** xref:tools/io-tool/onboarding.adoc[Onboarding]
+*** xref:tools/io-tool/logging.adoc[Logging]
+*** xref:tools/io-tool/endpointList.adoc[Endpoints]
+*** xref:tools/io-tool/filesending.adoc[File Sending]
+** xref:tools/io-tool.adoc[IO-Tool]
```

Comment:
There's currently 2 IO-Tool Entries here, move this to Line 63 and delete Line 63